pkgs/development/web/mailcatcher/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, lib, bundlerEnv, ruby, makeWrapper }:
+
+stdenv.mkDerivation rec {
+  name = "mailcatcher-${version}";
+
+  version = (import ./gemset.nix).mailcatcher.version;
+
+  env = bundlerEnv {
+    name = "${name}-gems";
+
+    inherit ruby;
+
+    gemdir = ./.;
+  };
+
+  buildInputs = [ makeWrapper ];
+
+  phases = ["installPhase"];
```

Comment:
I agree with @joachifm here, don't set phases explicitly. We typically only set it in generic builders but not in arbitrary packages.